### DISCUSSION TITLE
Pantone Color Names?

### DISCUSSION THREAD

**Discussion by bytrangle** [meodai/color-names#147] [Q&A] [answered]
Hi. I wonder if it's possible to find the closest matching Pantone color and return the Pantone color name. There is no API for looking up a Pantone color name from its code as far as I can tell. Or is it not possible because Pantone color names are copyrighted?

**Comment by meodai** [comment, marked as answer]
No unfortunately Pantone is a proprietary system. There are quite a few repositories that have them. https://github.com/Margaret2/pantone-colors I just don't think they are compatible with an open source license.

**Reply by bytrangle** [reply]
Thanks for the answer. I've tried using the query `goodnamesonly=true` and find the results to be quite unpredictable. For example, given the HEX code #4e8f00, omitting the query returns "Pistachio flour" while adding it returns "Pesto Alla Genovese". 

I perceive pesto color as gray green, while pistachio is yellowish green. Besides, I'm thinking of using `color-names`to return an intuitive color name (not sure if that's the right word) for a product with many color variations. Where I live, pistachio is much more popular than the exotic pesto so the word "pistachio" conveys the color better.

I think it will be more helpful to be able to specify a naming source in the API.

**Reply by meodai** [reply]
It is. It's not documented in this repository, because I am in the process of moving the API part to its own repository. https://github.com/meodai/color-name-api . Instead of `goodnamesonly`, you can now specify a list. Please not that it only contains lists that should be compatible with an open-source license. Hope it helps

**Reply by meodai** [reply]
PS: used it here for the list on the very bottom for example https://codepen.io/meodai/full/PoaRgLm

**Reply by bytrangle** [reply]
Cool. Thank you so much for your amazing work.

**Reply by meodai** [reply]
@bytrangle PS: feel free to submit changes for the colors you think are not as fitting as they should be
